proposal/service.go

```diff
@@ -80,9 +80,11 @@ func (s *Service) List(opts ListOpts) ([]v2.Proposal, error) {
 		return values(resultMap), nil
 	}
 
-	for k, proposal := range resultMap {
```

Comment:
Probably not much difference but you dont even need to call the above `.. := s.qualityService.Sessions(opts.from)` if `includeMonitoringFailed` is false 🤷🏻